### PR TITLE
Add tw1t611/ german keyboard layout for minidox.

### PR DESCRIPTION
**Pull request by tw1t611**
## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/minidox/keymaps/tw1t611/keymap.c [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+// fillers to make layering more clear
+#define _______ KC_TRNS
```
Comment (drashna) [inline keyboards/minidox/keymaps/tw1t611/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/minidox/keymaps/tw1t611/keymap.c [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#define XXXXXXX KC_NO
```
Comment (drashna) [inline keyboards/minidox/keymaps/tw1t611/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/minidox/keymaps/tw1t611/keymap.c [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+),
+
+[_ADJUST] = LAYOUT( \
+  _______,  _______,  _______,  _______,  _______,             _______,   KC_F7,    KC_F8,    KC_F9,   KC_F10,  \
+  CALTESC,  CALTDEL,  _______,  CALT,     _______,             _______,   KC_F4,    KC_F5,    KC_F6,   KC_F11,  \
+  RESET,    _______,  _______,  _______,  _______,             _______,   KC_F1,     KC_F2,    KC_F3,   KC_F12,  \
+                          _______,  _______,  _______,      _______,  _______,  _______  \
+)};
+
+void persistant_default_layer_set(uint16_t default_layer) {
```
Comment (drashna) [inline keyboards/minidox/keymaps/tw1t611/keymap.c]: It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Comment (tw1t611) [reply keyboards/minidox/keymaps/tw1t611/keymap.c]: I'll fix that, thanks for your nice response.

**Comment by tw1t611** [comment]
hi, i changed the layout. Is there a reason, why it wasn't merged last time?

**Comment by drashna** [comment]
Thanks! 

And I'm not sure what you mean.

**Pull request merged**